test/spec/modules/kimberliteBidAdapter_spec.js

```diff
@@ -0,0 +1,167 @@
+import { spec } from 'modules/kimberliteBidAdapter.js';
+import { assert } from 'chai';
+import { deepClone, randomIdentifierStr } from '../../../src/utils';
+import { BANNER } from '../../../src/mediaTypes.js';
+
+const BIDDER_CODE = 'kimberlite';
+
+describe('kimberliteBidAdapter', function () {
+  const sizes = [[640, 480]];
+
+  describe('isBidRequestValid', function () {
+    let validBid = {
+      mediaTypes: {
+        [BANNER]: {
+          sizes: [[320, 240]]
+        }
+      },
+      params: {
+        placementId: 'test-placement'
+      }
+    };
+
+    it('pass on valid bidRequest', function () {
+      assert.isTrue(spec.isBidRequestValid(validBid));
+    });
+
+    it('fails on missed placementId', function () {
+      let bid = deepClone(validBid);
```

Comment:
instead of doing a deepClone each time you could just have a `beforeEach` which sets the `validBid` object each time.

Then you just can alter it however you want in your test, not worrying about it effecting downstream tests